CONTRIBUTING.md

```diff
@@ -281,11 +282,21 @@ If you want to run a spell check of all `.Rmd` files locally, you can use run `R
 
 ## Rendering notebooks
 
+### Automatic rendering using GitHub actions
+
 This repository uses [snakemake](https://snakemake.readthedocs.io/en/stable/getting_started/installation.html) to render all notebooks.
+All notebooks are automatically re-rendered by GitHub actions before merges to master.
+The newly rendered html files are all pushed to the `gh-pages` branch which will publish the material to https://alexslemonade.github.io/refinebio-examples/.
+
+If this automatic rendering fails, you can see the errors on GitHub by going to `Actions` and clicking the workflow of PR you are working on that also says `Build Docker` underneath.
```

Comment:
Two things here:
1. How will they know it has failed? (I think they should get an email, but have we tested this? Do we get an email when merge automations fail?)
2. What do they do with a failure? I would encourage them to make changes immediately to fix the error, and or contact maintainers for support.

Reply:
I get emails when things I push failed, but depending on peoples' notification settings I could see that this isn't necessarily a given? Anyway, I've adjusted this wording, see what you think,